crates/gateway-config/src/mcp.rs

```diff
@@ -0,0 +1,27 @@
+use serde_dynamic_string::DynamicString;
+
+#[derive(Clone, Debug, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
```

Comment:
You could have added `#[serde(default)]` and implemented `Default`, find it clearer imho.

Reply:
There is no default for things like name tho...

Reply:
ah ok. Would have used a default name like `Grafbase GraphQL MCP`or something like this imho. Feels like a nice-to-have to have a good name, rather than something absolutely required to get it up and running.

Reply:
good defaults is always a good idea to reduce friction